.changeset/tsc.md

```diff
@@ -0,0 +1,32 @@
+---
+"@biomejs/js-api": fix
+---
+
+Biome JavaScript Bindings has now specific
+[subpaths exports](https://nodejs.org/api/packages.html#subpath-exports) for the
+three packages:
+
+- `import { Biome } from "@biomejs/js-api/bundler";`
+- `import { Biome } from "@biomejs/js-api/nodejs";`
+- `import { Biome } from "@biomejs/js-api/web";`
+
+These new exports load only TypeScript declarations, whereas the default export*
+loads declarations for all three packages. This was a problem if you checked
+your code with
+[`tsc`](https://www.typescriptlang.org/docs/handbook/compiler-options.html).
```

Comment:
I'm not sure I understand this paragraph. What's `export*`? Also, if we mentioned the default export, shouldn't we provide an example or something?

Reply:
I used the term [_subpath export_](https://nodejs.org/api/packages.html#subpath-exports) to avoid confusion with [_export_](https://developer.mozilla.org/Web/JavaScript/Reference/Statements/export). The `*` in `export*` was a typo.